analytics/migrations/0002_auto_20170828_1105.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-08-28 11:05
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('analytics', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='analyticsrequestsrecord',
+            name='date',
+            field=models.DateField(default=datetime.date.today, unique=True, verbose_name=b'Record Date'),
```

Comment:
why there is a migration when there is no modification in `analytics` model in this PR?